config/config.go

```diff
@@ -35,10 +35,11 @@ type Metrics struct {
 
 // Counter is a metric defining prometheus counter
 type Counter struct {
-	Name   string  `yaml:"name"`
-	Help   string  `yaml:"help"`
-	Table  string  `yaml:"table"`
-	Labels []Label `yaml:"labels"`
+	Name    string  `yaml:"name"`
+	Help    string  `yaml:"help"`
+	Table   string  `yaml:"table"`
+	PerfMap string  `yaml:"perf_map"`
```

Comment:
Let's document this in `README.md` as well. It's probably worth mentioning this in `counter` section and link to a new section describing the use case.

Reply:
Done in a bit more concise way. Would that suffice?